sport/app/football/model/CompetitionStages.scala

```diff
@@ -20,7 +20,7 @@ class CompetitionStage(competitions: Seq[Competition]) {
       competition: Competition,
       orderings: Map[String, List[ZonedDateTime]] = Map.empty,
   ): List[CompetitionStageLike] = {
-    val stagesWithMatches = competition.matches.toList.groupBy(_.stage).toList
+    val stagesWithMatches = competition.matches.groupBy(_.stage).toList.sortBy(_._1.stageNumber.toIntOption)
```

Comment:
The code previously relied on `groupBy` producing a Map that had the keys in the 'natural' order of stage 1, stage 2 - but didn't have any sorting to enforce that. Maps are unordered collections, and so there was never a guarantee that Scala 2.13 will put Map keys in the same order as Scala 2.12.